#121 Add basic doc structure using docusaurus (#125)
clay/claycli: website/README.md

You will need to to have `Node >= 8.x` and `Yarn >= 1.5`.

# Run the local server

1. Make sure all the dependencies for the website are installed:

```sh
$ yarn
```
or
```sh
$ npm install
```

> 2. Go to the `webside` directory and run your dev server:

Reviewer: Typo in here, please fix it
```suggestion
2. Go to the `website` directory and run your dev server:
```

Author: Thanks!! and done!!!

```sh
$ yarn start
```
or
```sh
$ npm run start
```

## Directory Structure

Your project file structure should look something like this

```
my-docusaurus/
  docs/
    cli.md
  website/
    blog/
    core/
    node_modules/
    pages/
    static/
      css/
      img/
    versioned_docs/
    versioned_sidebars/
    README.md
    package.json
    sidebar.json
    siteConfig.js
    versions.json
```

# Publish the website

> The Claycli webside will live on GitHub page. You just need to run the following commands:

Reviewer: Same suggestion as before
```suggestion
The Claycli website will live on GitHub page. You just need to run the following commands:
```

Author: 😅

```
$ yarn build
```
or
```
$ npm run build
```

> Then you just need to deploy the statics files generated. This files will be pushed to the `gh-page` branch.

Reviewer: I think `statics` should be singular
```suggestion
Then you just need to deploy the static files generated. These files will be pushed to the `gh-page` branch.
```
